openwisp_monitoring/check/tests/test_iperf3.py

```diff
@@ -317,32 +318,36 @@ def test_iperf3_check_config(self, mock_warn, mock_exec_command):
     @patch.object(iperf3_logger, 'warning')
     def test_iperf3_device_connection(self, mock_warn):
         dc = self.dc
-        with self.subTest('Test active device connection when management tunnel down'):
-            with patch.object(
-                device_connection, 'connect', return_value=False
-            ) as mocked_connect:
-                self._perform_iperf3_check()
-                mock_warn.assert_called_with(
-                    f'DeviceConnection for "{self.device}" is not working, iperf3 check skipped!'
-                )
-            self.assertEqual(mocked_connect.call_count, 1)
```

Comment:
I don't think that we should remove this test.

Reply:
Ok, I'll revert this change, thanks :+1: